Thinktecture.Relay.Server/Controller/ManagementWeb/UserController.cs

```diff
@@ -48,6 +51,18 @@ public IHttpActionResult Create(CreateUser user)
 				return BadRequest();
 			}
 
+			// new password and repetition need to match
```

Comment:
These checks (and the following in `Update()` should be moved to a method to reuse them.